erlang/api_cowboy/.gitignore

```diff
@@ -0,0 +1,21 @@
+.rebar3
+_build
+_checkouts
+_vendor
+.eunit
+*.o
+*.beam
+*.plt
+*.swp
+*.swo
+.erlang.cookie
+ebin
+log
+erl_crash.dump
+.rebar
+logs
+.idea
+*.iml
+rebar3.crashdump
+*~
+*.DS_Store
```

Comment:
You should enable the setting `Files: Insert Final Newline` in VSCode to get rid of those files without a new line at EOF.